api/managementpb/ia/rules.proto

```diff
@@ -0,0 +1,123 @@
+syntax = "proto3";
+
+package ia.v1beta1;
+
+option go_package = "iav1beta1";
+
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/duration.proto";
+
+// TODO move it to parent directory / package once API is v1-stable
+
+// ParamUnit Integrated Alerting rule parameter unit.
+enum ParamUnit {
+  PARAM_UNIT_INVALID = 0;
+  PERCENTAGE = 1;
+}
+
+// ParamType Integrated Alerting rule parameter type.
+enum ParamType {
+  PARAM_TYPE_INVALID = 0;
+  FLOAT = 1;
+}
+
+// Rule represents Integrated Alerting rule.
+message Rule {
+  // Param repsesents a single Integrated Alerting rule parameter.
+  message Param {
+    // Parameter name.
+    string name = 1;
+    // Parameter description.
+    string help = 2;
+    // Parameter unit.
+    ParamUnit unit = 3;
+    // Parameter type.
+    ParamType type = 4;
+    // Parameter minimal value (float).
+    float min_value = 5;
+    // Parameter maximum value (float).
+    float max_value = 6;
+    // Parameter set value (float).
+    float default_value = 7;
+    // Parameter set value (float).
+    float value = 8;
+  }
+  // Rule name.
+  string name = 1;
+  // Rules status: enabled or disabled.
+  bool enabled = 2;
+  // Rule description.
+  string help = 3;
+  // Rule parameters.
+  repeated Param params = 4;
+  // Rule default duration.
+  google.protobuf.Duration default_for = 5;
+  // Rule set duration.
+  google.protobuf.Duration for = 6;
+}
+
+message ListAlertingRulesRequest {
+  // If true, Integrated Alerting rule files will be re-read from disk.
+  bool reload = 1;
+}
+
+message ListAlertingRulesResponse {
+  // Integrated Alerting rules.
+  repeated Rule rules = 1;
+}
+
+message ChangeAlertingRulesRequest {
+  // Param repsesents a single Integrated Alerting rule parameter change.
+  message Param {
+    // Parameter name.
+    string name = 1 [
+      (validator.field) = {
+        string_not_empty: true
+      }
+    ];
+    // Parameter set value (float).
+    float value = 2;
+    // Remove set value (reset to default).
+    bool remove_value = 3;
+  }
+  // Rule name.
+  string name = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // Sets new rule status: enabled or disabled. Must always be sent.
+  // FIXME ?
+  bool enabled = 2;
```

Comment:
@BupycHuk WDYT?

Reply:
it's a little bit different from what we have now and it's better to discuss with a team. With frontend devs as well. I understand that we should change our logic and don't pass two fields instead of one, but not sure how we will handle two different approaches and keep backward compatibility in the same time.